js/form.js

```diff
@@ -2,6 +2,151 @@ const formContainer = document.querySelector('.ad-form');
 const formFieldsets = formContainer.querySelectorAll('fieldset');
 const filtersContainer = document.querySelector('.map__filters');
 const filtersFieldsets = filtersContainer.querySelectorAll('fieldset');
+const roomsContainer = formContainer.querySelector('#room_number');
+const guestsContainer = formContainer.querySelector('#capacity');
+const typeContainer = formContainer.querySelector('#type');
+const priceContainer = formContainer.querySelector('#price');
+const timeIn = formContainer.querySelector('#timein');
+const timeOut = formContainer.querySelector('#timeout');
+const TitleContainer = formContainer.querySelector('#title');
+
+const TYPES = {
+  bungalow: 0,
+  flat: 1000,
+  hotel: 3000,
+  house: 5000,
+  palace: 10000
+};
+
+const CAPACITY = {
+  maxRooms: 100,
+  nonGuests: 0
+};
+
+const TITILE_LENGTH = {
+  maxlength: 100,
+  minlength: 30
+};
+
+const maxPrice = 100000;
+
+const pristine = new Pristine(formContainer, {
+  classTo: 'ad-form__element',
+  successClass: 'ad-form__element--valid',
+  errorClass: 'ad-form__element--invalid',
+  errorTextParent: 'ad-form__element',
+  errorTextTag: 'span',
+  errorTextClass: 'ad-form__error-text',
+});
+
+// Валидация заголовка объявления
+
+function validateTitle (value) {
+  return value.length >= TITILE_LENGTH.minlength && value.length <= TITILE_LENGTH.maxlength;
+}
+
+function getErrorTitle () {
+  if (TitleContainer.value.length < TITILE_LENGTH.minlength) {
+    return ' Количество символов должно быть больше 30';
+  } else if (TitleContainer.value.length > TITILE_LENGTH.maxlength) {
+    return ' Количество символов должно быть меньше 100';
+  }
+}
+
+// Сопоставление типов жилищ с минимальной ценой и сопоставление времени выезда и заезда
+
+function syncTypesWithMinPrice () {
+  const type = typeContainer.value;
+
+  if (type === 'flat') {
+    priceContainer.placeholder = TYPES.flat;
+  }
+  if (type === 'palace') {
+    priceContainer.placeholder = TYPES.palace;
+  }
+  if (type === 'bungalow') {
+    priceContainer.placeholder = TYPES.bungalow;
+  }
+  if (type === 'hotel') {
+    priceContainer.placeholder = TYPES.hotel;
+  }
+  if (type === 'house') {
+    priceContainer.placeholder = TYPES.house;
+  }
+}
+
+function syncTimesOut () {
+  timeIn.value = timeOut.value;
+}
+
+function syncTimesIn () {
+  timeOut.value = timeIn.value;
+}
+
+timeOut.addEventListener('change', syncTimesOut);
```

Comment:
во всех eventListener второй аргумент именуется on+где+что случилось - посмотри критерии
значит твою функицю syncTimesOut надо всунуть внутрь функции с нормальным названием
```suggestion
timeOut.addEventListener('change', onTimeInChange);
```